src/redux/categories/categories.js

```diff
@@ -0,0 +1,28 @@
+import { createReducer } from '@reduxjs/toolkit';
+import Categories from '../../components/pages/Categories';
+
+// Define action type for addding & removing
+const CHECK_STATUS = 'bookstore/categories/CHECK_STATUS';
+const CREATE_CATEGORIES = 'bookstore/Categories/CREATE_CATEGORIES';
+
+// set intial state to empty array
+const initialState = [];
+
+export const checkStatus = () => ({
+  type: CHECK_STATUS,
+});
+
+export const createCategories = () => ({
+  type: CREATE_CATEGORIES,
+  payload: Categories,
+});
+
+// Action creator
+const categoriesReducer = createReducer(initialState, {
+  [CHECK_STATUS]: () => 'Under construction',
+  [CREATE_CATEGORIES]: (state, action) => {
+    state.push(...action.payload);
```

Comment:
:red_circle:  Kindly make sure you return the state as requested by the previous reviewer. The return keyword stops the execution of a function and returns the desired output.

**NOTE**: Because your arrow functions use curly brackets, {} you cannot implicitly return. To have an implicit return please make sure the function body is an expression, not a block